service/src/test/java/uk/gov/hmcts/reform/fpl/config/utils/ElementUtilsTest.java

```diff
@@ -0,0 +1,233 @@
+package uk.gov.hmcts.reform.fpl.config.utils;
+
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+import uk.gov.hmcts.reform.fpl.model.common.Element;
+import uk.gov.hmcts.reform.fpl.model.common.dynamic.DynamicList;
+import uk.gov.hmcts.reform.fpl.model.common.dynamic.DynamicListElement;
+
+import java.util.List;
+import java.util.Optional;
+import java.util.UUID;
+import java.util.function.Function;
+
+import static java.util.Arrays.asList;
+import static java.util.Collections.emptyList;
+import static java.util.UUID.randomUUID;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static uk.gov.hmcts.reform.fpl.utils.ElementUtils.asDynamicList;
+import static uk.gov.hmcts.reform.fpl.utils.ElementUtils.findElement;
+import static uk.gov.hmcts.reform.fpl.utils.ElementUtils.unwrapElements;
+import static uk.gov.hmcts.reform.fpl.utils.ElementUtils.wrapElements;
+
+public class ElementUtilsTest {
```

Comment:
Good that we are added tests for ElementUtils now but it is a bit out of scope of PR, perhaps should have been in separate PR (should try to avoid inflating PR with other code changes - but I agree this is valuable). Just note for future

Reply:
Most of added tests are about functionality which has been added to ElementUtils due to placement needs so in this PR